go/src/koding/klientctl/team.go

```diff
@@ -3,15 +3,63 @@ package main
 import (
 	"encoding/json"
 	"fmt"
-	"koding/klientctl/endpoint/team"
 	"os"
+	"text/tabwriter"
+
+	"koding/kites/kloud/team"
+	epteam "koding/klientctl/endpoint/team"
 
 	"github.com/codegangsta/cli"
 	"github.com/koding/logging"
 )
 
+func TeamList(c *cli.Context, log logging.Logger, _ string) (int, error) {
+	opts := &epteam.ListOptions{
+		Slug: c.String("slug"),
+	}
+
+	teams, err := epteam.List(opts)
+	if err != nil {
+		return 0, err
+	}
+
+	if len(teams) == 0 {
```

Comment:
It doesn't return both arguments nil (see [here](https://github.com/koding/koding/pull/9852/files#diff-725382ef8210b3747b2451c5ce354d47R79) ). This mean that we get error only on real failures. Empty list is not an error here it's just a normal result from kloud call. It's Kloud that handles NotFound errors from db and not KD.